examples/quickstart.ipynb

```diff
@@ -42,15 +42,20 @@
       "source": [
         "%%capture\n",
         "#@title Install required packages. (Run Cell)\n",
-        "! rm -r ./Mava\n",
-        "! git clone https://github.com/instadeepai/Mava.git\n",
-        "!pip install ./Mava[reverb,jax,launchpad,envs]\n",
-        "\n",
-        "# Installs for agent visualisation.\n",
-        "!pip install ./Mava[record_episode]\n",
-        "! apt-get update -y &&  apt-get install -y xvfb &&  apt-get install -y python-opengl && apt-get install ffmpeg && pip install pyvirtualdisplay \n",
-        "# Google colab has an old version of cloudpickle - issue with lp.  \n",
-        "!pip install cloudpickle -U"
+        "# Fix common bug in installing 'box-2d'\n",
+        "!apt-get install swig\n",
+        "\n",
+        "# Get Mava\n",
+        "%pip install \"id-mava[reverb,jax,envs] @ git+https://github.com/instadeepai/mava.git\"\n",
+        "\n",
+        "# Visualisation stuff\n",
+        "!apt-get update -y &&  apt-get install -y xvfb &&  apt-get install -y python-opengl && apt-get install ffmpeg && pip install pyvirtualdisplay \n",
+        "\n",
+        "# Colab has an old version of cloudpickle\n",
+        "%pip install cloudpickle -U\n",
+        "\n",
+        "# Colab has an old/conflicting numpy version\n",
+        "%pip install numpy"
```

Comment:
So I did some digging on the `module compiled against API version 0xf but this version of numpy is 0xe` error and it seems like our jaxlib version is compiled for another version of numpy. (our jaxlib version is compiled for 1.22.x (https://github.com/numpy/numpy/blob/f8e87f8d5892dd5fe11ddd05470ae83a0104d417/numpy/core/setup_common.py#L46), while colab has np 1.21.x . 

Can you try changing the setup.py to set our numpy version (`numpy==1.22.4`) and then run this cell with `pip install numpy` removed?

We shouldn't ever see the `module compiled...` error.

Reply:
Thanks, @KaleabTessera!

Firstly—apologies for making the commit (https://github.com/instadeepai/Mava/pull/865/commits/9b75e1731c25b153489a00c33bdfd694b88861bd) in order to _test_ whether it works. This is bad practice, but I am not sure how else one can debug with changes _within_ Colab? I can't test locally and then push, because the problem seems to be with Colab itself. (I suppose I could try manually uploading the repo to Colab's workspace 🤔)

Anyway, I tried setting the numpy version as suggested, but a different version is loaded beforehand:
```
WARNING: The following packages were previously imported in this runtime:
  [dataclasses,enum,google,numpy]
You must restart the runtime in order to use newly installed versions.
```

Ignoring the warning, the old `numpy` is used—so we see the same `module compiled...` error. 

Of course, following the advice to restart the runtime fixes the issue, but its frustrating: forcing the user to restart their runtime to continue with the notebook. Doing it in code also feels hacky.

Thoughts?

Reply:
No worries about the extra commit! It is tricky debugging the quickstart. 

Agreed, doing it in code feels hacky. The user can restart manually. If you remove the `pip install numpy`, then I think this is ready to go.

Reply:
Okay cool! One more problem: when the output is `%%capture`d, the prompt to restart the runtime doesn't show. But even when it shows, it's hidden halfway through a wall of text 🥴  I'll add an instruction to the user to restart the runtime manually, at least as an interim solution? 👍🏻

Reply:
Completed in https://github.com/instadeepai/Mava/pull/865/commits/813a4fe6cf18eb024445f1bbaffbc3f6711e4c6e :)

Reply:
Great, thanks @callumtilbury !